xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-api/src/main/resources/ApplicationResources_de.properties

```diff
@@ -17,5 +17,28 @@
 # Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 # 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 # ---------------------------------------------------------------------------
-entitynamevalidation.create.invalidname=Dokument {0} kann nicht erstellt werden, da der Name die Namensstrategie des Wikis nicht respektiert. \u00C4ndern Sie den Namen oder verschieben Sie das \u00FCbergeordnete Element in einen anderen Bereich.
+### Missing: entitynamevalidation.create.invalidNameError=Cannot create page {0} because its name does not respect the configured name strategy of the wiki.
+### Missing: entitynamevalidation.create.invalidName.possibleSolution=Change the name or move the parent to another space.
+### Missing: entitynamevalidation.create.invalidName.possibleSolutionForLink=Edit the link of current page so that it complies with the current name strategy.
+### Missing: entitynamevalidation.create.invalidName.strategyInfo=The configured name strategy is: {0}. The strategy is defined as follows: {1}
+# ---------------------------------------------------------------------------
+# See the NOTICE file distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+# ---------------------------------------------------------------------------
```

Comment:
Wrong comment

Reply:
Yeah, the usual "this used to be the first key in the file".